### PR TITLE
fix: specify timestamp type in where condition

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

specify timestamp type in where condition

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/v0.5/zh/user-guide/query-data/sql.md [hunk 1]
```diff
@@ -111,38 +111,38 @@ SELECT * FROM monitor WHERE host='127.0.0.1' AND ts > '2022-11-03 03:39:57';
...
+你可以使用 `::TimestampSecond` 语法来指定它的类型为 `TimestampSecond`，
+这样可以确保数据库正确地识别 `1667446797` 是一个以秒为单位的时间戳。
 
 ```sql
-SELECT * from monitor WHERE ts >= now() - INTERVAL '5 minutes';
+select * from monitor where ts > 1667446797::TimestampSecond;
 ```
 
-<!-- Use the `arrow_cast` function to cast the number literal `1650252336408` (Unix Epoch `2022-04-18 03:25:36.408` with millisecond resolution) to the timestamp type with millisecond precision:
+对于标准的 `RFC3339` 或 `ISO8601` 字符串，将它们指定为 `TimestampMillisecond` 类型。
```
Comment (waynexia) [inline docs/v0.5/zh/user-guide/query-data/sql.md:132]:
```suggestion
对于标准的 `RFC3339` 或 `ISO8601` 字符串格式，它们在精度上没有歧义，所以通常不需要指定时间戳类型。
```

File: docs/v0.5/zh/user-guide/query-data/sql.md [hunk 2]
```diff
@@ -111,38 +111,38 @@ SELECT * FROM monitor WHERE host='127.0.0.1' AND ts > '2022-11-03 03:39:57';
...
 SELECT * FROM monitor WHERE ts > 1667446797000;
 ```
 
-你还可以使用时间函数来过滤数据。
-<!-- Here are some examples of using time and date functions in the `WHERE` clause. -->
-例如，使用 `now()` 函数和 `INTERVAL` 关键字来获取最近 5 分钟的数据：
+当过滤条件中的时间值类型与时间索引列的值类型不同时，
+你需要使用 `::` 语法来指定时间值的类型来确保数据库正确的识别时间值的类型。
+例如数字 `1667446797` 表示一个以秒为单位的时间戳，
+你可以使用 `::TimestampSecond` 语法来指定它的类型为 `TimestampSecond`，
```
Comment (waynexia) [inline docs/v0.5/zh/user-guide/query-data/sql.md:125]: Should we list all available options here? The full set is:

- second: "TIMESTAMP_S" | "TIMESTAMP_SEC" | "TIMESTAMPSECOND"
- millisecond: "TIMESTAMP_MS" | "TIMESTAMPMILLISECOND"
- microsecond: "TIMESTAMP_US" | "TIMESTAMPMICROSECOND"
- nanosecond: "TIMESTAMP_NS" | "TIMESTAMPNANOSECOND"

If this is too verbose, we can only give the first short type

**Review by waynexia — APPROVED** [review]
Others LGTM 👍

File: docs/v0.5/zh/user-guide/query-data/sql.md (hunk 2 above)
Comment (nicecui) [reply docs/v0.5/zh/user-guide/query-data/sql.md:125]: I will write these content to the Data-types doc.

**Review by sunng87 — APPROVED** [review]

**Pull request merged**